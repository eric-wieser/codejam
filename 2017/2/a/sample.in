5
4 3
4 5 6 4
4 2
4 5 6 4
3 3
1 1 1
5 4
1 1 1 1 3
7 4
1 1 1 1 3 2 2
