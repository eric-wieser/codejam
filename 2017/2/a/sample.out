[4, 5, 6, 4] 3
Case #1: None
[4, 5, 6, 4] 2
Case #2: None
[1, 1, 1] 3
Case #3: None
