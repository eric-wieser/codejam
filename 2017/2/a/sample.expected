Case #1: 3
Case #2: 4
Case #3: 1
Case #4: 4